test/parallel/test-https-set-timeout-server.js

```diff
@@ -21,6 +21,7 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: can you please move this after the `common.hasCrypto` check?

Reply:
Done.